doc/classes/Viewport.xml

```diff
@@ -286,6 +286,12 @@
 		<member name="use_xr" type="bool" setter="set_use_xr" getter="is_using_xr" default="false">
 			If [code]true[/code], the viewport will use the primary XR interface to render XR output. When applicable this can result in a stereoscopic image and the resulting render being output to a headset.
 		</member>
+		<member name="vrs_mode" type="int" setter="set_vrs_mode" getter="get_vrs_mode" enum="Viewport.VRSMode" default="0">
+			The VRS mode that is used for this viewport. Note, if hardware does not support VRS this property is ignored.
```

Comment:
At least here it should likely spell out "Variable Rate Shading (VRS) mode" once.

Reply:
I put it in a few strategic places so people don't have to go looking for it :)